fiftyone/operators/store/service.py

```
@@ -1,44 +1,43 @@
+        repo (None): a
+            :class:`fiftyone.factory.repos.execution_store.ExecutionStoreRepo`
+        dataset_id (None): a dataset ID to scope operations to
     """
-    svc = ExecutionStoreService(dataset_id=dataset_id)
-    svc.cleanup_for_dataset()
 
-
-class ExecutionStoreService(object):
     def __init__(
         self,
         repo: Optional["ExecutionStoreRepo"] = None,
```

Ruff

Undefined name `ExecutionStoreRepo`

(F821)